README.md

```diff
@@ -166,349 +49,31 @@ Preact supports modern browsers and IE9+:
-
-Components also have a `render()` method, but unlike React this method is passed `(props, state)` as arguments. This provides an ergonomic means to destructure `props` and `state` into local variables to be referenced from JSX.
-
-Let's take a look at a very simple `Clock` component, which shows the current time.
-
-```js
-import { h, render, Component } from 'preact';
-
-class Clock extends Component {
-	render() {
-		let time = new Date();
-		return <time datetime={time.toISOString()}>{ time.toLocaleTimeString() }</time>;
-	}
-}
-
-// render an instance of Clock into <body>:
-render(<Clock />, document.body);
-```
 
+const App = () => {
+	const [input, setInput] = useState('');
 
-That's great. Running this produces the following HTML DOM structure:
-
-```html
-<span>10:28:57 PM</span>
-```
-
-In order to have the clock's time update every second, we need to know when `<Clock>` gets mounted to the DOM. _If you've used HTML5 Custom Elements, this is similar to the `attachedCallback` and `detachedCallback` lifecycle methods._ Preact invokes the following lifecycle methods if they are defined for a Component:
-
-| Lifecycle method            | When it gets called                              |
-|-----------------------------|--------------------------------------------------|
-| `componentWillMount`        | before the component gets mounted to the DOM     |
-| `componentDidMount`         | after the component gets mounted to the DOM      |
-| `componentWillUnmount`      | prior to removal from the DOM                    |
-| `componentWillReceiveProps` | before new props get accepted                    |
-| `shouldComponentUpdate`     | before `render()`. Return `false` to skip render |
-| `componentWillUpdate`       | before `render()`                                |
-| `componentDidUpdate`        | after `render()`                                 |
-
-
-
-So, we want to have a 1-second timer start once the Component gets added to the DOM, and stop if it is removed. We'll create the timer and store a reference to it in `componentDidMount()`, and stop the timer in `componentWillUnmount()`. On each timer tick, we'll update the component's `state` object with a new time value. Doing this will automatically re-render the component.
-
-```js
-import { h, render, Component } from 'preact';
-
-class Clock extends Component {
-	constructor() {
-		super();
-		// set initial time:
-		this.state = {
-			time: Date.now()
-		};
-	}
-
-	componentDidMount() {
-		// update time every second
-		this.timer = setInterval(() => {
-			this.setState({ time: Date.now() });
-		}, 1000);
-	}
-
-	componentWillUnmount() {
-		// stop when not renderable
-		clearInterval(this.timer);
-	}
-
-	render(props, state) {
-		let time = new Date(state.time).toLocaleTimeString();
-		return <span>{ time }</span>;
-	}
+	return (
+		<div>
+			<p>Do you agree to the statement: "Preact is awesome"?</p>
+			<input value={input} onChange={e => setInput(e.target.value)} />
```

Comment:
Well I think it's pretty linear to just exposing them to doing this in a `Component`, no?

Reply:
Since the community is moving toward Hooks right now, I think making the first example Hooks driven gives additional confidence the author written code will not need to be significantly change or refactored.

Reply:
+1 For a hooks driven example. My gut tells me that users coming from React will perceive Preact as "old" when the first example only shows classes.